modules/issue_tracker/ajax/ToggleMyWatching.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Issue tracker
+ *
+ * Toggles the issue-watching flag for a user.
+ *
+ * PHP Version 5
+ *
+ * @category Loris
+ * @package  Issue_Tracker
+ * @author   Caitrin Armstrong <caitrin.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+/**
+ * Issue tracker
+ *
+ * Toggles the issue-watching flag for a user.
+ *
+ * PHP Version 5
+ *
+ * @category Loris
+ * @package  Issue_Tracker
+ * @author   Caitrin Armstrong <caitrin.mcin@gmail.com>
```

Comment:
@AnyhowStep You can put your name here, since you wrote the whole script, or put a generic `Loris Team <loris.mni@bic.mni.mcgill.ca>`